src/hct_mis_api/apps/payment/models.py

```diff
@@ -431,9 +444,49 @@ def delivery_mechanism(self) -> Optional[str]:
         return self.payment_plan.delivery_mechanisms.first().delivery_mechanism
 
 
-class PaymentPlan(ConcurrencyModel, SoftDeletableModel, GenericPaymentPlan, UnicefIdentifiedModel, AdminUrlMixin):
+class PaymentPlan(
+    ConcurrencyModel,
+    SoftDeletableModel,
+    GenericPaymentPlan,
+    UnicefIdentifiedModel,
+    AdminUrlMixin,
+    InternalDataFieldModel,
+):
+    TP_MIGRATION_MAPPING = {
+        # tp.field: payment_plan.field
+        # if value has internal_data__ will story into json
+        "name": "name",
+        "ca_id": "internal_data__ca_id",
+        "ca_hash_id": "internal_data__ca_hash_id",
+        "created_by": "created_by",
+        "change_date": "status_date",
+        "business_area": "business_area",
+        "status": "status",
+        "build_status": "build_status",
+        "built_at": "built_at",
+        "program": "program",
+        "program_cycle": "program_cycle",
+        "targeting_criteria": "targeting_criteria",
+        "sent_to_datahub": "internal_data__sent_to_datahub",
+        "steficon_rule": "steficon_rule",
```

Comment:
we need 2 separate fields for steficon rules. One for targeting steficon rule (calculate vulnerabilities), second for payment plan rule (calculate entitlement)

Reply:
will add `steficon_rule_targeting`, `steficon_targeting_applied_date`